examples/limitedLanguageList.html

```diff
@@ -7,19 +7,13 @@
 
 		<title>Universal Language Selector</title>
 		<meta name="author" content="Santhosh Thottingal">
-		<link href="../css/jquery.uls.css" rel="stylesheet">
-		<link href="../css/jquery.uls.grid.css" rel="stylesheet">
-		<link href="../css/jquery.uls.lcd.css" rel="stylesheet">
+		<link href="../styles/jquery.uls.css" rel="stylesheet">
 		<!-- demo -->
 		<link href="resources/demo.css" rel="stylesheet">
 		<!-- Libs -->
 		<script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
 		<!-- Source -->
-		<script src="../src/jquery.uls.data.js"></script>
-		<script src="../src/jquery.uls.data.utils.js"></script>
-		<script src="../src/jquery.uls.lcd.js"></script>
-		<script src="../src/jquery.uls.languagefilter.js"></script>
-		<script src="../src/jquery.uls.core.js"></script>
+		<script src="../jquery.uls.js"></script>
```

Comment:
Running the examples directly from the root `examples` directory no longer works. We should update the README.md file to mention that the users should build the files, and then run the examples from the `dist/examples` folder.

Reply:
Once this PR is done, I plan to create a static application running from dist folder in github pages or netlify or similar places. Along with that I will address this confusion